cmd/dex/serve.go

```diff
@@ -193,6 +202,8 @@ func runServe(options serveOptions) error {
 	if err != nil {
 		return fmt.Errorf("failed to initialize storage: %v", err)
 	}
+	defer closeStorage(logger, s)
```

Comment:
I kinda think `defer s.Close()`  should be enough for now. Failure to close a storage is not actionable, we can't do anything with the fact that it failed.

Reply:
Oh, ok. I will simplify it.

Incorrectly closing storage can affect the database, e.g., the connection may be left open. I thought this message can help with debugging problems on the storage side.